packages/neo-one-client-core/src/user/RemoteReceiveUserAccountProvider.ts

```diff
@@ -0,0 +1,70 @@
+import { Block, RawAction, UserAccountProvider } from '@neo-one/client-common';
+import { Endpoint, Message } from './messageTypes';
+
+export interface RemoteReceiveUserAccountProviderOptions {
+  readonly endpoint: Endpoint;
+  readonly userAccountProvider: UserAccountProvider;
+}
+
+export class RemoteReceiveUserAccountProvider {
+  private readonly messageEndpoint: Endpoint;
```

Comment:
Remove all of these instance members, you don't need them, just make local variables.